Lib/test/test_new_pyc.py

```diff
@@ -93,7 +93,7 @@ def test_consts(self):
     def do_test_speed(self, body, call=False):
         NUM_FUNCS = 100
         functions = [
-            f"def f{num}(a, b):\n{body}"
+            f"def f{num}(a, b):\n{body}".replace("a", f"a{num}_").replace("b", f"b{num}_")
```

Comment:
The replace() call bothers me a bit, since it's so context-free. Maybe we should invest a little in a better way of generating N functions with M lines each following a pattern that's a function of N and M?

Reply:
Agree. We probably want both tests, unique and not unique, to measure with and without the interning benefit.